ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -144,6 +143,8 @@ class BigDiffy[T : Coder](@transient val lhs: SCollection[T], @transient val rhs
   private lazy val globalAndFieldStats: SCollection[(GlobalStats, Iterable[FieldStats])] =
     BigDiffy.computeGlobalAndFieldStats(_deltas, ignoreNan)
 
+
+  implicit val deltasCoder: Coder[(MultiKey, String, Any, Any)] = Coder.kryo
```

Comment:
We should add a comment here about using kryo. This might have to change in a future version